### PR TITLE
readme: update WeChat contact info

### PR DESCRIPTION
**Pull request by dcalvin**
## What have you changed? (mandatory)
- Updated WeChat request info

## What are the type of the changes? (mandatory)
- Update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Join the TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invit
 
 ### WeChat
 
-The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personal information (WeChat ID, and other basic information) included. We will invite you in right away.
+The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to zhangyanqing@pingcap.com, with your personal information (WeChat ID, PR you submitted if any, and other basic information) included. We will invite you in right away.
```
Comment (siddontang) [inline README.md]: seem we must acquire the submitted PR

Comment (dcalvin) [reply README.md]: You mean it's mandatory, right?

Comment (siddontang) [reply README.md]: yes, right

File: README.md [hunk 2]
```diff
@@ -205,7 +205,13 @@ Join the TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invit
...
 ### WeChat
 
-The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personal information (WeChat ID, and other basic information) included. We will invite you in right away.
+The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to zhangyanqing@pingcap.com, with your personal information that includes the following:
+
+- WeChat ID (**Required**)
+- PR you submitted to TiKV Repos (**Required**)
+- Other basic information
+
+We will invite you in right away.
```
Comment (dcalvin) [inline README.md]: @gingerkidney

File: README.md [hunk 3]
```diff
@@ -205,7 +205,13 @@ Join the TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invit
 
 ### WeChat
 
-The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personal information (WeChat ID, and other basic information) included. We will invite you in right away.
+The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to zhangyanqing@pingcap.com, with your personal information that includes the following:
```
Comment (Hoverbear) [inline README.md]: Please link `zhangyanqing@pingcap.com` to `mailto:zhangyanqing@pingcap.com`

File: README.md [hunk 4]
```diff
@@ -205,7 +205,13 @@ Join the TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invit
 
 ### WeChat
 
-The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personal information (WeChat ID, and other basic information) included. We will invite you in right away.
+The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to mailto:zhangyanqing@pingcap.com, with your personal information that includes the following:
```
Comment (Hoverbear) [inline README.md]:
```suggestion
The TiKV community is also available on WeChat, a very popular messaging and social media application in China. If you want to join our WeChat group, send a request mail to [zhangyanqing@pingcap.com](mailto:zhangyanqing@pingcap.com), with your personal information that includes the following:
```

**Comment by dcalvin** [comment]
HI @siddontang  Please can you merge the PR. Thanks.

**Comment by gingerkidney** [comment]
LGTM

**Comment by siddontang** [comment]
Thanks @gingerkidney

Please notice that the applicants must provide the TiKV pull request links.

**Pull request merged**